main.go

```diff
@@ -45,6 +47,10 @@ func main() {
 		"use format 'example-ns1/example-service1' to enable a single service, use format 'example-ns2/*' to enable all services in a namespace, and use '*' to enable all services in the cluster' ")
 	combinationPolicyTag := flag.String("combo-policy-tag", "proxy-principals", "key of tag for proxy principals list")
 	authPolicyControllerOnlyMode := flag.Bool("auth-policy-only-mode", false, "only run authzpolicy controller")
+	enableSpiffeTrustDomain := flag.Bool("enable-spiffe-trust-domain", true, "Allow new SPIFFE ID's")
+	adminDomain := flag.String("admin-domain", "k8s.omega.stage", "admin domain")
```

Comment:
Considering this is opensource repo, we should not have any internal-related information exposed. Can we use something else as a default value?

Reply:
We can pass in the values via flag, i left it empty as default value